crates/future/src/lib.rs

```diff
@@ -258,15 +230,7 @@ pub trait StaticFutureExt {
     /// #     example().await;
     /// # }
     /// ```
-    #[cfg(feature = "metrics")]
-    fn spawn(self, name: &'static str) -> JoinHandle<<Self::Future as Future>::Output>;
-
-    /// Same as [`StaticFutureExt::spawn`], but it won't monitor long running
-    /// futures.
-    ///
-    /// Use this only if your future is expected to be long running (ex.
-    /// singleton).
-    fn spawn_and_forget(self) -> JoinHandle<<Self::Future as Future>::Output>;
+    fn spawn(self) -> JoinHandle<<Self::Future as Future>::Output>;
```

Comment:
Can we return the labels? It was very convenient to set up metrics to monitor for leaks.

Reply:
Otherwise there's really no point in having `spawn()` method at all, as usually it's more convenient to use `tokio::spawn()`.

Reply:
Makes sense.